testData/solutions/win32.json

```diff
@@ -12,13 +12,49 @@
             {
                 "type": "gpii.settingsHandlers.INISettingsHandler",
                 "options": {
-                    "filename": "${{environment}.APPDATA}\\Freedom Scientific\\JAWS\\15.0\\Settings\\VoiceProfiles.ini"
+                    "filename": "${{environment}.APPDATA}\\Freedom Scientific\\JAWS\\16.0\\Settings\\enu\\DEFAULT.JCF",
```

Comment:
Thanks. I have documented the settings files on the wiki page [Screen Reader Settings: JAWS](https://wiki.gpii.net/w/Screen_Reader_Settings:_JAWS).